sputnikdao-factory2/src/factory_manager.rs

```diff
@@ -0,0 +1,167 @@
+//! Module for standard generic contract factory manager.
+
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::json_types::Base58CryptoHash;
+use near_sdk::serde_json;
+use near_sdk::{env, sys, AccountId, Balance, CryptoHash, Gas};
+
+/// Gas spent on the call & account creation.
+const CREATE_CALL_GAS: Gas = Gas(75_000_000_000_000);
+
+/// Gas allocated on the callback.
+const ON_CREATE_CALL_GAS: Gas = Gas(10_000_000_000_000);
+
+/// Leftover gas after creating promise and calling update.
+const GAS_UPDATE_LEFTOVER: Gas = Gas(20_000_000_000_000);
+
+const NO_DEPOSIT: Balance = 0;
+
+/// Factory manager that allows to store/load contracts by hash directly in the storage.
+/// Uses directly underlying host functions to not load any of the data into WASM memory.
+#[derive(BorshSerialize, BorshDeserialize)]
+pub struct FactoryManager {}
+
+impl FactoryManager {
+    /// Store contract from input.
+    pub fn store_contract(&self) {
```

Comment:
`store_contract` is a bit vague.

What about:
```
/// Store a new version of the DAO contract as WASM code.
/// The DAO contract will be saved as a key-value pair inside the storage.
/// The key is represented by the `sha256` representation of the WASM code.
/// The value is represented by the WASM code itself.
pub fn store_dao_contract_code(&self) {}
pub fn store_wasm_code_for_dao_contract(&self) {}
```

Insisting on specifying that the contract code is for the DAO, because at first, I thought it's the contract code for the factory itself.

Reply:
nope, this is generic library - will move to near sdk later